index.css

```diff
@@ -0,0 +1,87 @@
+header {
+  position: absolute;
+  display: flex;
+  width: 1440px;
+  height: 72px;
+  background-color: #3e3c3c;
+}
+a { text-decoration: none; }
+
+.oluchi {
+  width: 500px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: white;
+}
+
+.port {
+  width: 80px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: white;
+}
+
+.about {
+  width: 45px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: white;
+}
+
+.contact {
+  width: 63px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: white;
+}
+
+.h {
+  color: #f55800;
+  width: 566px;
+  height: 160px;
+  line-height: 82px;
+}
+
+section {
+  display: flex;
+  padding: 50px;
+  flex-direction: column;
+  width: 566px;
+  height: 739px;
+  align-items: center;
+}
+
+main {
+  position: absolute;
+  width: 1440px;
+  height: 870px;
+  left: 0;
+  top: 0;
+  background: #1c1a19;
+}
+
+aside {
+  position: absolute;
+  left: 600px;
+  top: 50px;
+  width: 600px;
+  transform: matrix(-0.96, 0.27, 0.27, 0.96, 0, 0);
+}
+
+img {
+  position: absolute;
+  width: 600px;
+  top: 0%;
+  bottom: 0%;
+  transform: matrix(-0.96, 0.27, 0.27, 0.96, 0, 0);
+}
+
+.container {
+  display: flex;
+  width: 1440px;
+  height: 870px;
+}
```

Comment:
- [ ] According to the requirements, You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). Remember that in this project you should work on a mobile version not desktop version 💯
- [ ]  The Header section should contain the logo and the mobile menu icon .
- [ ] The Headline section should contain the main title, sub-title, paragraph, social icons, and background, It should be  similar as much as possible to your Figma design 👌

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/191674152-bbd37e44-bc8c-47c2-9d57-d15a1ca7f637.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/191674061-2a09d148-263b-4ffb-937f-aeec8b3cbe52.png" width="500" /> |